packages/evolution-frontend/src/styles/survey/_admin.scss

```
@@ -47,6 +47,45 @@
+
+        ._trip-diary-invalid-message {
+            // Leave room for the invalidate button in the top-right corner
+            margin: 0 2rem 0.5rem 0;
+        }
+
+        ._member-trip-diary-invalidate {
+            position: absolute;
+            top: 0.25rem;
+            right: 0.25rem;
+
+            padding: 0.15rem 0.4rem;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)